docs/guides/dao-members/proposals.md

```diff
@@ -269,18 +248,19 @@ any change you'll make to your proposal will change its hash.
 
 <img src="../assets/images/ipfs-proposals.png" width="550"/>
 
-To host a proposal description on IPFS:
+There are hosting services for IPFS such as [Pinata<ExternalLinkImage/>]() and
+[Fleek<ExternalLinkImage/>]() to name a few. To host a proposal description on
```

Comment:
Did you mean to add links for these?

Reply:
Yah that is a silly mistake. I will fix and merge. Thanks for the catch.